.gitignore

```diff
@@ -14,6 +14,12 @@ manifest.json
 # conflicts with FB internal infra
 .watchmanconfig
 
+# Don't checkout the unnecessary yarn/npm lock files
+yarn.lock
```

Comment:
Ah, I think this is what confused me. Could you specialise this to only refer to `./yarn.lock`? Then we don't accidentally exclude future real lock files that we DO want to check in. :)

Reply:
ahhh. I remember doing that only initially (relative path), but for some reason vscode did NOT *un-highlight* the lock file (suggesting us that it's ignored by git) so added direct path and it worked, that could be due to some issue with vscode's internal git plugin, since I knew `.gitignore` supports (prefers) relative paths.

Thanks. Pushing now...